src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+  const propertyValues = Object.values(robot);
+  let current;
+
+  for (let i = 0; i < propertyValues.length; i++) {
+    current = propertyValues[i];
+
+    for (let j = i + 1; j < propertyValues.length; j++) {
+      if (current === propertyValues[j]) {
+        return null;
+      }
+    }
+  }
+
+  for (const key in robot) {
+    inverse[robot[key]] = key;
+  }
```

Comment:
please create variable for robot[key]
```suggestion
  for (const key in robot) {
    if(inverse has already this property) { don't add another key-value pair }
  
    inverse[robot[key]] = key;
  }
```